zwave_js/Dockerfile

```
@@ -18,7 +18,7 @@ RUN \
         python3 \
     \
     && npm config set unsafe-perm \
-    && npm install -g @zwave-js/server@${ZWAVEJS_SERVER_VERSION} zwave-js@${ZWAVEJS_VERSION} \
+    && npm install -g @zwave-js/server@${ZWAVEJS_SERVER_VERSION} zwave-js@${ZWAVEJS_VERSION} zwave-js/core@${ZWAVEJS_VERSION} \
```

There was an `@` missing.
```suggestion
    && npm install -g \
        @zwave-js/server@${ZWAVEJS_SERVER_VERSION} \
        zwave-js@${ZWAVEJS_VERSION} \
        @zwave-js/core@${ZWAVEJS_VERSION} \
```